fenix/app/src/main/java/org/mozilla/fenix/compose/tabstray/TabGridItem.kt

```diff
@@ -131,6 +139,13 @@ fun TabGridItem(
                     .then(tabBorderModifier)
                     .padding(4.dp)
                     .combinedClickable(
+                        interactionSource = interactionSource,
+                        indication = rememberRipple(
+                            color = when (isSystemInDarkTheme()) {
+                                true -> PhotonColors.White
+                                false -> PhotonColors.Black
```

Comment:
I think the underlying question is do we even need to override the color here? does it provide a default value?

Reply:
The default value provided by Android would be [black](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/Indication.kt;l=159) for both light and dark themes. From my discussion with Verdi it was clear we want the tab to get brighter for light theme at least.

cc @Verdi

Reply:
gotcha gotcha. I think this is because we don't provide our own `RippleTheme`. I'd say for now, let's pass in a color directly and file a follow-up to write [our own custom override](https://stackoverflow.com/questions/67265345/modify-ripple-color-of-iconbutton-in-jetpack-compose) for our `FirefoxTheme`.

Reply:
Opened [1840020](https://bugzilla.mozilla.org/show_bug.cgi?id=1840020) in this scope.

Reply:
This sounds like the right idea, thanks!